templates/terraform/examples/billing_budget_basic.tf.erb

```diff
@@ -0,0 +1,19 @@
+data "google_billing_account" "account" {
+  display_name = "My Billing Account"
+  open         = true
+}
+
+resource "google_billing_budget" "<%= ctx[:primary_resource_id] %>" {
```

Comment:
could you also add a second example/test that exercises more of the available api options?

Reply:
Done.